mt-batcher/services/common/crypto.go

```diff
@@ -96,9 +98,9 @@ func ParsePrivateKeyStr(privKeyStr string) (*ecdsa.PrivateKey, error) {
 	return crypto.HexToECDSA(hex)
 }
 
-func ParseWalletPrivKeyAndContractAddr(name string, mnemonic string, hdPath string, privKeyStr string, contractAddrStr string) (*ecdsa.PrivateKey, common.Address, error) {
+func ParseWalletPrivKeyAndContractAddr(name string, mnemonic string, hdPath string, privKeyStr string, contractAddrStr string, password string) (*ecdsa.PrivateKey, common.Address, error) {
 
-	privKey, err := GetConfiguredPrivateKey(mnemonic, hdPath, privKeyStr)
+	privKey, err := GetConfiguredPrivateKey(mnemonic, hdPath, privKeyStr, password)
```

Comment:
How about adding a empty checker for `password`?

Reply:
The password can be passed empty without restriction